faqs.html

```diff
@@ -13,6 +13,18 @@ <h1 class="text-accent-4 center">Frequently Asked Questions</h1>
       <div class="collapsible-header black clemson-orange-text">When is the event?</div>
       <div class="collapsible-body"><span>CUCTF is taking place October 3rd, 2020.</span></div>
     </li>
+    <li>
+      <div class="collapsible-header black clemson-orange-text">COVID-19?</div>
+      <div class="collapsible-body">
+      <div class="card-content"> <span class="card-title clem"><i class="material-icons">error_outline</i></span>
+        <p>CU Cyber is aware of the lingering uncertainty with the COVID-19 pandemic. Our hearts are with those affected by the virus, and our leadership team is taking the steps necessary to ensure CUCTF 2020 is up and operational come October 3rd. </p>
```

Comment:
Make it clear here safety is number 1 and operational event is number 2. Also probably rephrase the first sentence to acknowledging COVID-19 is posing an increasing safety risk given the recent uptick in cases. (Since I mean obviously we are aware of the ongoing pandemic and in general it is uncertain but it is currently posing an increasing problem -- cases recorded by DHEC have doubled for Clemson's zip code in the last few days and I doubt that number is going to go down once the semester starts.)